www/userguide/userguide.md

```diff
@@ -0,0 +1,290 @@
+- [Report navigation tree](#userguide-report-navigation-tree)
+- [Button pane](#userguide-button-pane)
+- [Bug path view](#userguide-bug-path-view)
+- [Comments](#userguide-comment)
+
+![Report view](images/report_view.png)
+
+## <a name="userguide-report-navigation-tree"></a> Report navigation tree
+Report Navigation Tree shows the found reports at the currently opened file. The
+reports are grouped by the severity level.
+You can navigate between them by clicking on a node in the tree.
+
+![Report navigation tree](images/report_navigation_tree.png)
+
+## <a name="userguide-button-pane"></a> Button pane
+Button Pane contains several items which help you to change or get some
+property of the currently opened report. 
+![Button pane](images/button_pane.png)
+
+### <a name="show-documentation"></a> Show documentation
+Show Documentation button shows the documentation of the actual checker which
+identified by the currently opened report.
+![Checker documentation](images/checker_documentation.png)
+
+### <a name="userguide-review-status"></a> Review status
+Reports can be assigned a review status of Unreviewed, Confirmed bug, False
+positive, Intentional, along with an optional comment on why this status was
+applied.
+- <span class="customIcon review-status-unreviewed"></span>  **Unreviewed**
+(_default_): Nobody has seen this report.
+- <span class="customIcon review-status-confirmed-bug"></span> **Confirmed**:
+This is really a bug.
+- <span class="customIcon review-status-false-positive"></span>
+**False positive**: This is not a bug.
+- <span class="customIcon review-status-intentional"></span> **Intentional**:
+This report is a bug but we don't want to fix it.
+
+We can change the review status from the default
+<span class="customIcon review-status-unreviewed"></span> _Unreviewed_ option
+to something else in the report details view above the file view.
+![Unreviewed](images/review_status_unreviewed.png)
+
+If you changed the review status, you can optionally explain the reason why
+you changed it.
+![Change review status](images/review_status_change.png)
+
+If somebody has already changed the review status from the default one, you can
+see extra information (who changed the review status, when and why) beside
+the review status selector by hovering on the message icon. This message icon is
+hidden by default if nobody has changed the review status.
+![Review status message](images/review_status_message.png)
+
+## <a name="userguide-same-reports"></a> Same reports
+Several reports may belong to a specific bug if the but itself can be reached on
+different control paths. In the Bug viewer you can check whether the selected
+bug is available on a different path.
+
+![Same reports](images/same_reports.png)
+
+## <a name="userguide-bug-path-view"></a> Bug path view
+Some checkers are able to follow the execution path along the control flow of
+the program. If a bug appears on any of these paths then CodeChecker is able to
```

Comment:
According to the spellchecker: "If your sentence begins with a condition, insert a comma in front of the independent clause."

So a `,` between paths and then.